docs/users-guide/06-dashboards.md

```diff
@@ -0,0 +1,88 @@
+
+Once you're in edit mode you'll see a grid appear. You can move and resize the cards in the dashboard to your liking and they'll snap to the grid.
+
+![Editing dashboard](images/dashboards/DashboardEdit.png)
+
+- To move cards just click and drag.
+- To resize a card just click and drag the handle at the bottom right corner of the card.
+- To remove a card, click the X icon in the top right corner.
+
+Questions in your dashboard will automatically update their display based on the size you choose to make sure your data looks great at any size.
+
+
+### Deleting a dashboard
+Deleting a dashboard does not delete the individual saved questions on it — it just deletes the dashboard. Remember — dashboards are shared by everyone on your team, so think twice before you delete something that someone else might be using!
+
+To delete a dashboard, click the pencil-looking **Edit** icon in the top right of the dashboard, then click **Delete**.
+
+### Fullscreen dashboards
+
+After you've made your ideal dashboard you may want to put it on a TV or present it in some other visible space to help keep your team up-to-date throughout the day.
+
+To enter fullscreen mode just click the fullscreen icon in the top right of the dashboard.
+
+Once you've entered fullscreen mode you can also switch the dashboard into "Night mode" for higher contrast.
+
+![Night mode](images/dashboards/DashboardNightMode.png)
+
+### Auto refresh
+If your data is more realtime in nature you can set your dashboard up to auto refresh.
+
+![Autorefresh](images/dashboards/DashboardAutorefresh.png)
+
+You can set your dashboard to update in 1, 5, 10, 15, 30, and 60 minute intervals depending on how fresh you need the data to be.
+
+Enabling auto refresh will re-run all the queries on the dashboard at the interval you choose, so keep the size of the dashboard and the complexity of the questions in mind when setting up auto refresh.
+
+Combining fullscreen mode and auto refresh is a great way to keep your team in sync with your data throughout the day.
+
+### Configuring a dashboard through its URL
+
+It is possible to amend URL of the dashboard to automatically enter fullscreen, enable night mode or auto-refresh the dashboard. This allows you to configure the dashboard even when you do not have any input access to the device where the dashboard will be displayed, like scripted screens, for example.
+
+To configure a dashboard using its URL, you can add the following optional keywords: `fullscreen`, `night`, or `refresh`. Here's an example:
+
+`https://metabase.mydomain.com/dash/2#refresh=60&fullscreen&night`
+
+The part that says `refresh=60` sets the dashboard to automatically refresh every 60 seconds, `fullscreen` sets it to fullscreen mode, and `night` sets it to night mode (night mode only works when using fullscreen). Use an `&` in between keywords, and make sure there's a hash `#` after the dashboard's ID number.
```

Comment:
@tlrobinson does this mean the refresh variable can be any integer number of seconds?

i.e. `https://metabase.mydomain.com/dash/2#refresh=1&fullscreen&night` would attempt to refresh every second?